src/nyc_trees/apps/core/views.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from datetime import datetime
+from calendar import timegm
+
+from django.conf import settings
+from django.shortcuts import render_to_response
+from django.utils.timezone import make_aware, utc
+
+from apps.survey.models import Blockface, Survey
+
+
+def js_settings(request):
+    blockface_updated_at = _get_last_updated_datetime(Blockface)
+    survey_updated_at = _get_last_updated_datetime(Survey)
+
+    max_timestamp = lambda *datetimes: timegm(max(*datetimes).utctimetuple())
+
+    return render_to_response('core/settings.js', {
```

Comment:
Is this something to provide settings dynamically to the tiler? Trying to make sense of what this code is responsible for.